index.js

```diff
@@ -5,6 +5,10 @@
 var restify = require('restify')
 var bufferize = require('./lib/bufferize')
 var version = require('./package.json').version
+var errors = require('./lib/error')
+
+var otherRoutes = /\/.+/
+var methods = [ 'get', 'put', 'post', 'del', 'head' ]
```

Comment:
With a starting semi-colon since we don't use them in this repo and it'd be weird shizzles if you don't. :)

```
;['get', 'put', 'post', 'del', 'head'].forEach(function() {
```